[2025-05-18T02:23:49.035+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509
[2025-05-18T02:23:49.035+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api'
[2025-05-18T02:23:49.063+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509
[2025-05-18T02:23:49.064+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api'
[2025-05-18T02:23:49.095+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:23:47.833860+00:00 [queued]>
[2025-05-18T02:23:49.101+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:23:47.833860+00:00 [queued]>
[2025-05-18T02:23:49.102+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-18T02:23:49.114+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_api> on 2025-05-18 02:23:47.833860+00:00
[2025-05-18T02:23:49.118+0000] {standard_task_runner.py:60} INFO - Started process 212 to run task
[2025-05-18T02:23:49.120+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'extract_api', 'manual__2025-05-18T02:23:47.833860+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpwek_aikp']
[2025-05-18T02:23:49.122+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extract_api
[2025-05-18T02:23:49.160+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509
[2025-05-18T02:23:49.160+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:23:47.833860+00:00/task_id=extract_api'
[2025-05-18T02:23:49.162+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:23:47.833860+00:00 [running]> on host 42f8508e9f2c
[2025-05-18T02:23:49.367+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='extract_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T02:23:47.833860+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T02:23:47.833860+00:00'
[2025-05-18T02:23:50.189+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T02:23:50.198+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=extract_api, execution_date=20250518T022347, start_date=20250518T022349, end_date=20250518T022350
[2025-05-18T02:23:50.230+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T02:23:50.253+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
